src/Jackett.Common/Definitions/hdforever.yml

```
@@ -57,7 +57,7 @@ settings:
       MULTi.ENGLISH: MULTi.ENGLISH
       VOSTFR: VOSTFR
       MULTi.VOSTFR: MULTi.VOSTFR
-  - name: vostfr
+  - name: VOSTFR
```

```suggestion
  - name: vostfr
```

This can stay lowercase

---

revert done